src/SprykerEco/Yves/Easycredit/Theme/default/components/molecules/easycredit-summary/easycredit-summary.twig

```diff
@@ -0,0 +1,22 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'easycredit-summary'
+} %}
+
+{% define data = {
+    title: 'EasyCredit Summary:',
+    linkTitle: 'EasyCredit Link',
+    interest: required,
+    url: required,
+    text: required
+} %}
+
+{% block body %}
+    <div class="box">
+        <div class="spacing-y">{{ data.title }}</div>
+        <div class="spacing-y"> {{ data.interest }}</div>
```

Comment:
What can be inside `data.interest`. If it's just a text without markup suggest to use `p` here.